corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java

```diff
@@ -750,7 +790,7 @@ public void onNext(CorfuStreamEntries results) {
             @Override
             public void onError(Throwable throwable) {
                 isError = true;
-                log.error("Subscriber hit error", throwable);
```

Comment:
why are we changing the log level to info in case of errors?

Reply:
we are primarily doing this for capturing changes - whenever there are failures like this - no change is actually performed so even though there was an attempt made there is no change done - since ERROR is higher level than WARN, I am reducing these non-critical errors to info since the console will capture the failures anyway.